app/models/bulkrax/csv_entry.rb

```diff
@@ -61,7 +62,7 @@ def build_metadata
       add_admin_set_id
       add_collections
       add_local
-
+      self.save
```

Comment:
why is this saving here? save on the entry is called after the full build is done

Reply:
I can't remember why I did this but I feel like it was something to do with testing for metadata updating correctly. I'll remove it and if we see any negative effects we can put it back